lightning/src/ln/channelmanager.rs

```diff
@@ -5907,14 +5943,15 @@ where
 							// enforce option_scid_alias then), and if the funding tx is ever
 							// un-confirmed we force-close the channel, ensuring short_to_chan_info
 							// is always consistent.
+							let mut short_to_chan_info = self.short_to_chan_info.write().unwrap();
```

Comment:
Still confused here by the what is the correct lock order between `short_to_chan_info` and `channel_state` one L5895

Reply:
See https://github.com/lightningdevkit/rust-lightning/pull/1639#discussion_r973794535